Magi/Clara, the Folk Witch/Clara, Matilde's Sister.md

```diff
@@ -71,15 +74,32 @@ Clara grew as a Folk Witch skilled in entrancing people and creature to do her b
 
 # Baree, the Wolf
 
+Baree is a magical wolf that leads the wolves in the forests north of Venice, she's very independent but a few years ago he took a little girl under his wing, this girl was Clara. From that moment she follows and protects her, Clara reminds her of one of her own young… killed by a group of hunting men when she was younger.
```

Comment:
she or he ? (she's very independent ... he took)

Reply:
Uff... I need to proof read a bit more... I'll fix these with a she, as it should be.